nifi-mock/src/main/java/org/apache/nifi/util/TestRunner.java

```diff
@@ -1069,6 +1069,29 @@ void assertAttributes(
      */
     void setEnvironmentVariableValue(String name, String value);
 
+    /**
+     * Returns the current value of the Context Parameter with the given name
+     *
+     * @param name the name of the Context Parameter whose value should be returned.
+     * @return the current value of the Context Parameter with the given name or <code>null</code> if no value is currently set
+     *
+     * @throws NullPointerException if the name is null
+     */
+    String getContextParameterValue(String name);
+
+    /**
+     * Sets the value of the Context Parameter with the given name to be the given value. This exposes the Context Parameter
+     * for use by the Expression Language (with the #{name} syntax).
+     *
+     * NOTE - this method is only for testing purposes.
+     *
+     * @param name the name of the variable to set
+     * @param value the value of the variable
+     *
+     * @throws NullPointerException if either the name or the value is null
+     */
+    void setContextParameterValue(String name, String value);
+
```

Comment:
```suggestion
    /**
     * Sets a parameter with the given name and value into the simulated Parameter Context. This makes available the parameter to the properties of the component being tested through parameter referencing such as <code>#{name}</code>.
     *
     * @param parameterName the name of the parameter to set
     * @param parameterValue the value of the parameter to set
     *
     * @throws NullPointerException if the name is null
     */
    void setParameterContextValue(final String parameterName, final String parameterValue);

```

It is possible to set a value with `null`